### ISSUE TITLE
Sidebar open/closing hard (after Fridays update)

### ISSUE THREAD

**Issue by DavydeVries** [a85/POSTMan-Chrome-Extension#8782]
**Is your feature request related to a problem? Please describe.**
I don't like that the **sidebar icon** (to show/hide) is moved to the right (in the bar at the bottom). Because I use it a lot. Especially if I'm debugging on specific request. I like to close the sidebar on a small MacBook 13" screen. Then I looked up what the keyboard shortcut was. and that is `CMD + \`. But I'm also using 1Password, what results in popping-up 1Password...

**Describe the solution you'd like**
Two options:
- Move the icon back
- Make keyboard shortcuts customisable

**Describe alternatives you've considered**
I tried to use the keyboard shortcut but it's already taken by another app (1Password).

**Comment by coditva** [comment]
@DavydeVries Thanks for writing in! I understand your concern around this. We had moved the shortcut to the right to be close to other toggles like "build-browse" and "single pane - two pane layout" etc.

But we can definitely think more about this and let you know of any updates. :)

**Comment by imankurpatel000** [comment]
I agree with @DavydeVries, I was looking for a way to hide the left panel on the left side of the window. I spent almost half an hour looking for and searching on google how to hide the left panel in Postman and I got to know that the icon is in the bottom right while reading this issue. If the users can't figure out how to hide the side panel easily I would consider that a huge design flaw. Hope you guys can put it somewhere on the left side.
